example/DataContext.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+
+export const DataContext = React.createContext({ data: [] });
+
+export const DataContextProvider = ({ children }) => {
+  const [data, setData] = React.useState({ loading: true, data: [] });
+  React.useState(() => {
```

Comment:
Shouldn't this be `useEffect` or am I missing something ?

Reply:
thats true